### PR TITLE
Mm/dt 1333

### PR DESCRIPTION
**Pull request by MarcMcIntosh**
## Context

Editor config for unsplash and oembed does not work with a trailing slash

## Checklist before requesting a review
- [ ] I hereby declare my code ready for review.
- [ ] If it is a critical feature, I have added tests.
- [ ] The CI is successful.
- [ ] If there could backward compatibility issues, it has been discussed and planned.

### CODE REVIEW

File: packages/manager/src/lib/addTrailingSlash.ts [hunk 1]
```diff
@@ -3,3 +3,7 @@ export const addTrailingSlash = (
 ): string | undefined => {
 	return url?.replace(/\/?$/, "/");
 };
+
+export const removeTrailingSlash = (url: string): string => {
```
Comment (bapmrl) [inline packages/manager/src/lib/addTrailingSlash.ts:7]: ⚠️ The filename is misleading. We could rename it to `trailingSlash.ts`.

File: packages/manager/test/lib-addTrailingSlash.test.ts [hunk 2]
```diff
@@ -1,12 +1,30 @@
-import { expect, it } from "vitest";
+import { expect, it, describe } from "vitest";
```
Comment (bapmrl) [inline packages/manager/test/lib-addTrailingSlash.test.ts]: ⚠️ The filename is misleading. We could rename it to `lib-trailingSlash.test.ts`.

File: packages/manager/test/lib-addTrailingSlash.test.ts [hunk 3]
```diff
@@ -1,12 +1,30 @@
...
+		expect(addTrailingSlash("https://prismic.io")).toBe("https://prismic.io/");
+		expect(addTrailingSlash("https://prismic.io/")).toBe("https://prismic.io/");
+	});
 });
 
-it("adds a single trailing slash to endpoints", async () => {
-	expect(addTrailingSlash("https://prismic.io")).toBe("https://prismic.io/");
-	expect(addTrailingSlash("https://prismic.io/")).toBe("https://prismic.io/");
+describe("removeTrailingSlash", () => {
+	it("will removed a trailing slash from a string", () => {
```
Comment (bapmrl) [inline packages/manager/test/lib-addTrailingSlash.test.ts]: 💡 There's a typo (extra "d" to "removed"). We could change the message to "removes a single trailing slash to endpoints" for consistency or "will remove a trailing slash from a string" for your suggestion.

File: packages/manager/test/lib-addTrailingSlash.test.ts [hunk 4]
```diff
@@ -1,12 +1,30 @@
...
+		expect(removeTrailingSlash("https://prismic.io/")).toBe(
+			"https://prismic.io",
+		);
+		expect(removeTrailingSlash("https://prismic.io")).toBe(
+			"https://prismic.io",
+		);
+
+		expect(removeTrailingSlash("foo/bar")).toBe("foo/bar");
```
Comment (bapmrl) [inline packages/manager/test/lib-addTrailingSlash.test.ts]: ❓ What's the purpose of this expectation? Isn't it the same thing as `expect(removeTrailingSlash("https://prismic.io")).toBe("https://prismic.io")`?

**Review by bapmrl — CHANGES_REQUESTED** [review]
⚠️ Why are we changing the `package-lock.json` file if this PR doesn't include changes to the `package.json` dependencies?

File: packages/manager/test/lib-addTrailingSlash.test.ts (hunk 4 above)
Comment (MarcMcIntosh) [reply packages/manager/test/lib-addTrailingSlash.test.ts]: no trailing slash... double checking i guess

**Review by bapmrl — APPROVED** [review]

**Pull request merged**